ERRATA.md

```diff
@@ -0,0 +1,13 @@
+# UFS Short-Range Weather Application
+# Errata for the v2.1.0 release 
+
+GitHub Repository: https://github.com/ufs-community/ufs-srweather-app
+
+Release date: Nov. 17, 2022
+Release tag for the v2.1.0: **<release-tag>** 
```

Comment:
Should `<release-tag>` be replaced with ufs-srw-v2.1.0?

Reply:
Thank you so much, updated! 
Also, extended the description of the documented fix in errata file as following:
_... corrections for the Documentation, Section on runnning UFS SRW on MacOS and generic Linux using standalone wrapper scripts for each task._

Reply:
Great, thank you very much for taking care of this!